### ISSUE TITLE
Investigate disabled states of vf-link

### ISSUE THREAD

**Issue by nikiforosk** [visual-framework/vf-core#1913]
- [ ] Do we use the disabled state of vf-link in our websites?
- [ ] Are there any accessibility recommendations (e.g. WCAG or ARIA guides) against disabled states of links?

We might decide to retire this variant if it fails accessibility standards and is not used often in our sites.

**Comment by sandykadam** [comment]
@nikiforosk disabled state eg is when there is no paging or the previous link is disabled then it is used.
e.g 


![Image](https://user-images.githubusercontent.com/1227241/228271204-b8b2a6c7-c507-4b93-916d-61714e22aec6.png)
